### PR TITLE
CF-303: swap the order of partition column to match Hive DDL

### PR DESCRIPTION
**Pull request by shintasmith**
Found out that I was inserting date to feed column, and feed to date column. Ooops.

### CODE REVIEW

File: usmu/src/main/resources/copy_to_entries.q [hunk 1]
```diff
@@ -24,12 +24,12 @@ CREATE external TABLE IF NOT EXISTS ${INPUT_TABLE} (
        eventtype string,
        tenantid string,
        region string,
-       feed string,
-       date string)
+       date string,
```
Comment (ChandraAddala) [inline usmu/src/main/resources/copy_to_entries.q]: May date column shouldnt be a string. Looks like there is a DATE type for hive. Should we look into that?

Comment (shintasmith) [inline usmu/src/main/resources/copy_to_entries.q]: I copied this from Greg's spike. @usnavi, what do you think?

Comment (usnavi) [inline usmu/src/main/resources/copy_to_entries.q]: Spark doesn't support hive DATE type yet.
https://spark.apache.org/docs/1.1.0/sql-programming-guide.html#unsupported-hive-functionality

**Comment by usnavi** [comment]
lgtm;  i'll let @ChandraAddala merge.

**Pull request merged**